File: HELLOPYTHON/day10/myselect.py
import cx_Oracle

def getCprice(s_name):
    conn = cx_Oracle.connect('python/python@localhost:1521/xe')
    cs = conn.cursor()
    rs = cs.execute("select cprice FROM stock where s_name='"+s_name+"' order by in_time")
    
    ret = []
    for record in rs:
        ret.append(record[0])
    
    cs.close()
    conn.close()
    return ret
    
arr = getCprice("삼성전자")
print(arr)

File: HELLOWEB2/src/com/naver/web/MySelect.java
package com.naver.web;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.Statement;

public class MySelect {
	public static void main(String[] args) throws Exception {
		
		String driver = "oracle.jdbc.driver.OracleDriver";
		String url = "jdbc:oracle:thin:@127.0.0.1:1521:xe";
		String sql = "select s_code,s_name,cprice,in_time from stock where s_name='»ï¼ºÀüÀÚ'";

		Class.forName(driver);
		Connection conn = DriverManager.getConnection(url, "python", "python");
		Statement stmt = conn.createStatement();
		ResultSet rs = stmt.executeQuery(sql);
		while (rs.next()) {
			String s_code = rs.getString("s_code");
			String s_name = rs.getString("s_name");
			String cprice = rs.getString("cprice");
			String in_time = rs.getString("in_time");
			
			System.out.print("s_code:"+s_code);
			System.out.print("s_name:"+s_name);
			System.out.print("cprice:"+cprice);
			System.out.println("in_time:"+in_time);

		}
		
		stmt.close();
		conn.close();

	}
}




